src/app/modules/shared/components/header/header.component.scss

```diff
@@ -0,0 +1,18 @@
+@import "variables";
+
+.preloader {
+  position: absolute;
+
+  ::ng-deep .mat-progress-bar-fill::after {
+    background-color: rgba(38, 70, 112, 0.4);
+  }
+
+  ::ng-deep .mat-progress-bar-buffer {
+    background: $white;
+  }
+
+  ::ng-deep.mat-progress-bar {
+    border-radius: 5px;
+  }
+
+}
```

Comment:
you could create separate spinner component(even if you are using material preloading component) for customization and reusability in other's components